pos_stock_lot_required/static/src/js/screens.js

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2019 Druidoo - Iván Todorovich
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+   
+odoo.define('pos_stock_lot_required.screens', function (require) {
+"use strict";
+
+var screens = require('point_of_sale.screens');
+
+screens.ActionpadWidget.include({
+    renderElement: function() {
+        var self = this;
+        this._super();
+        this.$('.pay').click(function(){
```

Comment:
This works, but what's happening really is that the original popup is shown when `this._super()` is called, and then replaced by yours.

It would be better to do this:

```suggestion
        this.$('.pay').off('click').click(function(){
```

That'll disable the original event handler, before adding yours